src/proof/alethe/alethe_post_processor.h

```diff
@@ -13,8 +13,8 @@
  * The module for processing proof nodes into Alethe proof nodes
  */
 
-#ifndef CVC4__PROOF__ALETHE_PROOF_PROCESSOR_H
-#define CVC4__PROOF__ALETHE_PROOF_PROCESSOR_H
+#ifndef CVC5__PROOF__ALETHE_PROOF_PROCESSOR_H
```

Comment:
Actually this is also missing an `ALETHE__`

Reply:
Ops. :) Fixed